lib/features/family/features/account/presentation/pages/us_personal_info_edit_page.dart

```diff
@@ -0,0 +1,358 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_stripe/flutter_stripe.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/core/logging/logging_service.dart';
+import 'package:givt_app/features/account_details/bloc/personal_info_edit_bloc.dart';
+import 'package:givt_app/features/account_details/pages/change_address_bottom_sheet.dart';
+import 'package:givt_app/features/account_details/pages/change_bank_details_bottom_sheet.dart';
+import 'package:givt_app/features/account_details/pages/change_email_address_bottom_sheet.dart';
+import 'package:givt_app/features/account_details/pages/change_phone_number_bottom_sheet.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/auth/pages/change_password_page.dart';
+import 'package:givt_app/features/family/app/pages.dart';
+import 'package:givt_app/features/family/features/auth/helpers/logout_helper.dart';
+import 'package:givt_app/features/family/shared/widgets/common_icons.dart';
+import 'package:givt_app/features/registration/cubit/stripe_cubit.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/shared/dialogs/dialogs.dart';
+import 'package:givt_app/shared/pages/gift_aid_page.dart';
+import 'package:givt_app/shared/widgets/parent_avatar.dart';
+import 'package:givt_app/utils/stripe_helper.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+
+class USPersonalInfoEditPage extends StatelessWidget {
+  const USPersonalInfoEditPage({this.navigatingFromFamily = false, super.key,});
+  final bool navigatingFromFamily;
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    final locals = context.l10n;
+    final user = context.watch<AuthCubit>().state.user;
+    final isUkUser = Country.unitedKingdomCodes().contains(user.country);
```

Comment:
If you want you could clean up this file by only keeping US code (and remove if statements for us etc)